handlers/postList.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/db"
+)
+
+type postList struct {
+	db db.DB
+}
+
+func newPostList() RouteHandler {
+	return &postList{
+		db: db.DynamoDB(),
+	}
+}
+
+func (g *postList) match(request events.APIGatewayProxyRequest) bool {
```

Comment:
Think this `g` would be a `p` (I'm assuming this is copid from `getList` or something